src/search_unused.rs

```diff
@@ -440,17 +453,25 @@ pub(crate) fn find_unused(
             .collect()
     };
 
-    // Keep a side-list of ignored dependencies (likely false positives).
-    let ignored = analysis
+    let meta = analysis
         .manifest
         .package
         .as_ref()
         .and_then(|package| package.metadata.as_ref())
-        .and_then(|meta| meta.cargo_machete.as_ref())
+        .and_then(|meta| meta.cargo_machete.as_ref());
+
+    // Keep a side-list of ignored dependencies (likely false positives).
+    let ignored = meta
         .map(|meta| meta.ignored.iter().collect::<HashSet<_>>())
         .unwrap_or_default();
 
-    let workspace_ignored: HashSet<_> = workspace_ignored.into_iter().collect();
+    // Keep a list of renamed dependencies
+    static NO_RENAMED: BTreeMap<Box<str>, Box<str>> = BTreeMap::new();
+    let renamed = meta.map(|meta| &meta.renamed).unwrap_or(&NO_RENAMED);
+
+    let workspace_ignored: HashSet<_> = std::mem::take(&mut workspace_metadata.ignored)
```

Comment:
I think since `workspace_metadata.ignored` isn't used anymore until the end, and `workspace_metadata` isn't mutably borrowed on its own, the `stdm::mem::take()` might be overkill, and you could use `workspace_metadata.ignored` immediately here?

```suggestion
    let workspace_ignored: HashSet<_> = workspace_metadata.ignored
```

Reply:
Fixed